epub33/fxl-a11y/index.html

```diff
@@ -260,8 +262,7 @@
 
 If there are a group of images in sequence, you only have to describe details in the first image. In the proceeding images only mention what has changed. W3 Tutorial on Image Groups
 
-Not all details are needed in writing alt-text for images, and what you do describe relies heavily on context. For example, if the image is described in the surrounding text, you only need to briefly describe it in the Alt-text. For more information on when, and how to describe you can go to the DIAGRAM centre, or AccessibilePublishing.ca. See resources for links.
-    
+Not all details are needed in writing alt-text for images, and what you do describe relies heavily on context. For example, if the image is described in the surrounding text, you only need to briefly describe it in the Alt-text. For more information on when, and how to describe you can go to the DIAGRAM centre, or AccessibilePublishing.ca. See resources for links.    
```

Comment:
I don't think alt-text should be capitalized in the second sentence.

Reply:
It's kind of jargony to call it "alt text", too (no hyphen needed).

Might be better to write it out as "alternative text" all the time, as the document flips between the two. That or put alt text in parentheses after the first use of "alternative text" to indicate it's an abbreviation.

Reply:
Fixed!